cosmos/bfhevm_777.json

```
@@ -0,0 +1,49 @@
+{
+    "rpc": "https://rpc-bfhevm.xyz:8443/",
+    "rest": "https://rest-testnet-bfhevm.xyz:4431/swagger/#/",
```

hey as it's a swagger endpoint, trying to verify REST connectivity was fail. 
```suggestion
    "rest": "https://rest-testnet-bfhevm.xyz:4431",
```